runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/config/StreamingEngineComputationConfigFetcher.java

```diff
@@ -80,24 +81,28 @@ private StreamingEngineComputationConfigFetcher(
       long globalConfigRefreshPeriodMillis,
       WorkUnitClient dataflowServiceClient,
       ScheduledExecutorService globalConfigRefresher,
+      boolean shouldPerformOutputSizeChecks,
```

Comment:
nit: name to match the experiment, shouldThrowOutputSizeExceptions?

Reply:
Done. I went with `shouldThrowExceptionsOnLargeOutput`.